app/javascript/react/components/MyReviewTile.js

```diff
@@ -3,11 +3,18 @@ import React from "react";
 const MyReviewTile = props => {
   let deleteButton;
   let deleteButtonId = `deleteButton${props.id}`;
+  let editButton;
+  let editButtonId = `editButton${props.id}`;
```

Comment:
Declaring these here seems unnecessary. You can accomplish the same thing by putting this in the normal button tags below.